app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -52,14 +52,9 @@ def pending_profile_policy
       @pending_profile_policy ||= PendingProfilePolicy.new(
         user: current_user,
         resolved_authn_context_result: resolved_authn_context_result,
-        biometric_comparison_requested: biometric_comparison_requested?,
       )
     end
 
-    def biometric_comparison_requested?
-      @authorize_form.biometric_comparison_requested?
```

Comment:
need to remove `OpenidConnectAuthorizeForm#biometric_comparison_requested?`

Reply:
whoops, thank you! ::palmslap::